dts/bindings/rtc/atmel,sam-rtc.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+description: Atmel SAM family RTC device
+
+compatible: "atmel,sam-rtc"
+
+include: rtc-device.yaml
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  peripheral-id:
+    type: int
+    description: peripheral ID
+    required: true
+
+  alarms-count:
+    default: 1
+
+  timestamps-count:
+    default: 0
```

Comment:
Please, document last two. You can check https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/ieee802154/atmel%2Crf2xx.yaml as reference.

Reply:
The documentation for these values are in the rtc-device.yaml file from which they are included :)

Reply:
Moved to devicetree